plugins/inputs/metric_streams_listener/README.md

```diff
@@ -0,0 +1,132 @@
+# Metric Streams Input Plugin
+
+The Metric Streams plugin is a service input plugin that
+listens for metrics sent via HTTP and performs the required
+processing for
+[Metric Streams from AWS](#troubleshooting-documentation).
+
+Metrics from Metric Streams are Base64 encoded JSON
+blocks that adhere to
+[AWS's request & response specifications](#troubleshooting-documentation).
+
+The Telegraf plugin must return the required response format
+as outlined in
+[the Firehose documentation](#troubleshooting-documentation).
```

Comment:
This looks like valuable developer documentation, but maybe something users don't need to worry about. What do you think about moving them into the code as comments in the section where you decode the blocks and write the response?

Reply:
True, I can include it in the code with the response logic.